src/vivarium_census_prl_synth_pop/components/observers.py

```diff
@@ -610,8 +610,6 @@ def get_observation(self, event: Event) -> pd.DataFrame:
             df_w2[col] = df_w2["simulant_id"].map(pop_full[col])
 
         # Tracked, US population to be dependents or get their SSNs borrowed
-        # TODO: This should only be the living population -- leaving for now because
```

Comment:
So we are leaving dead simulants listed as dependents, and they are eligible to have their SSNs borrowed?

Reply:
oops, no, my mistake.